src/le.py

```diff
@@ -1212,7 +1221,7 @@ def _read_log_lines(self):
             buff_lines.append(self._read_file_rest[:MAX_BLOCK_SIZE])
             self._read_file_rest = self._read_file_rest[MAX_BLOCK_SIZE:]
 
-        return [line.decode('utf-8', 'ignore') for line in buff_lines[:-1]]
```

Comment:
@jsouthern-r7 

I'm looking at the changes to line 1224, of le.py, and I'm not sure that it will work in it's current form.
https://github.com/jyeung-r7/le/blob/python-2-3-compat/src/le.py#L1224
Currently, when the file, where the buff_lines come from, is opened, it's in text mode (the default).
https://github.com/jyeung-r7/le/blob/python-2-3-compat/src/le.py#L1160
Reading from it, in this case, will return String objects. Which will fail, when the codecs.decode() attempts to decode them. However, if the file is opened in binary mode, then it will return objects of type 'bytes', which can be decoded, using the currently implemented approach (codecs.decode()).
We'll want to make sure that this change is implemented, during the conversion.